test/benchmarks/Benchmarks.Trace/ElasticsearchBenchmark.cs

```diff
@@ -70,6 +75,24 @@ public int CallElasticsearchAsync()
 
             return task.GetAwaiter().GetResult();
         }
+
+        [Benchmark]
+        public unsafe object CallTargetCallElasticsearch()
+        {
+            return CallTarget.Run<RequestPipeline_CallElasticsearch_Integration, RequestPipeline, RequestData, object>(Pipeline, Data, &GetData);
+
+            static object GetData(RequestData data) => null;
+        }
+
+
+        [Benchmark]
+        public unsafe object CallTargetCallElasticsearchAsync()
```

Comment:
You should make sure to have the same return type as the callsite counterpart (here, `CallElasticsearchAsync` has a `int` return type) otherwise the comparison is unfair

Reply:
Right!, done in e06c511